addons/account/models/account_move.py

```diff
@@ -866,21 +872,19 @@ def _compute_suitable_journal_ids(self):
     @api.depends('posted_before', 'state', 'journal_id', 'date', 'move_type', 'origin_payment_id')
     def _compute_name(self):
         self = self.sorted(lambda m: (m.date, m.ref or '', m._origin.id))
-
         for move in self:
             if move.state == 'cancel':
                 continue
 
             move_has_name = move.name and move.name != '/'
-            if move_has_name or move.state != 'posted':
+            if move_has_name or move.state != 'posted' and move._get_last_sequence():
```

Comment:
Do you mean the method is not required anywhere anymore or just not here ?

In both cases, don't I still need something like it to detect whether this is the first move of the sequence or not ? I'm not aware of similar methods/fields/tools.

Reply:
Just here

> don't I still need something like it to detect whether this is the first move of the sequence or not ? 

Why would you? Maybe I'm forgetting something?
We needed to detect if it was the first because the first was set even in draft; but it's not the case anymore.